composer.json

```diff
@@ -37,6 +37,9 @@
         "silverstripe/standards": "^1",
         "phpstan/extension-installer": "^1.3"
     },
+    "conflict": {
+        "egulias/email-validator": "<3.2.6"
+    },
```

Comment:
Needed to fix https://github.com/silverstripe/silverstripe-contentreview/actions/runs/10447581437/job/29026521584?pr=250
> 1) SilverStripe\ContentReview\Tests\ContentReviewNotificationTest::testContentReviewEmails
> Symfony\Component\Mime\Exception\RfcComplianceException: Email "no-reply@localhost" does not comply with addr-spec of RFC 2822.

prefer-lowest is installing v3.1.0. Didn't go as far as to track the exact version of that library that fixes the bug - tried 3.1.2 and 3.2.0 and 3.2.6, the latter of which works as expected.